src/main/java/dev/lu15/voicechat/VoiceChat.java

```diff
@@ -51,12 +51,59 @@ sealed interface Builder permits VoiceChatImpl.BuilderImpl {
          */
         @NotNull Builder publicAddress(@NotNull String publicAddress);
 
+        /**
+         * Set the mtu of the voice server. This is used to determine the largest size of a packet.
+         * By default, this is set to 1024.
+         * @param mtu the public address of the voice server
+         * @return this builder
+         */
+        @NotNull Builder mtu(Integer mtu);
+
+        /**
+         * Set the codec of the voice server. This is used by clients to determine which audio codec to use.
+         * By default, this is set to VOIP.
+         * @param codec the voice codec to be used by clients
+         * @return this builder
+         */
+        @NotNull Builder codec(Codec codec);
+
+        /**
+         * Set the voice distance of the server. This is used to determine how far users can hear each other from..
+         * By default, this is set to 48.
+         * @param distance the distance players can hear each other from.
+         * @return this builder
+         */
+        @NotNull Builder distance(Integer distance);
+
+        /**
+         * Enables/Disables the use of groups on the server.
+         * By default, this is set to false.
+         * @param enabled used to determine if groups should be enabled.
+         * @return this builder
+         */
+        @NotNull Builder groups(Boolean enabled);
+
+        /**
+         * Set the keepalive delay for the server.
+         * By default, this is set to 1000.
+         * @param keepalive used to determine what the keepalive delay should be set to.
+         * @return this builder
+         */
+        @NotNull Builder keepalive(Integer keepalive);
+
+        /**
+         * Enables/Disables the use of recording on the server.
+         * By default, this is set to false.
+         * @param enabled used to determine if recording should be enabled.
+         * @return this builder
+         */
+        @NotNull Builder recording(Boolean enabled);
```

Comment:
Don't know if I missed this from the previous review, these should have primitive parameters rather than their boxed equivalents.